docs/user/docs/sessions/jupyter/collaboration.md

```diff
@@ -0,0 +1,62 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Collaboration with Jupyter Notebooks
+
+Collaborating on Jupyter notebooks is a common requirement in various
+workflows. This guide describes two main approaches to collaboration:
+individual sessions and project-level collaboration.
+
+## Collaboration in Individual Sessions
+
+If you have a notebook in your personal workspace and another user wants to
+connect to your sessions, follow these steps:
```

Comment:
This requires a newer (yet unreleased) version of capella-dockerimages, which is not pulled in by this PR as submodule update. I'm not sure what the release cycle here normally is – should this be mentioned somewhere, or will it just fall into place once everything gets merged together?

Reply:
Rule 1: Submodules are never up-to-date :D 

Joking aside, I'll update the submodule tomorrow and add the commit to this PR. In general, it doesn't affect the release process at all as sessions have their own lifecycle and the version has to be specified in the frontend for the production deployments.

Reply:
Update 1: I've created a new release in the capella-dockerimages repository.

Reply:
Update 2: Submodule is also updated.